### PR TITLE
Finalized DAEA Project

### PR DESCRIPTION
**Pull request by CareoWin**
Finalized DAEA Project on Kom Abu Bello by Caroline Nguyen

### CODE REVIEW

**Comment by jabiggs13** [comment]
I just went to your site and noticed that none of image popovers are working. When you click on them, it sends you right back up to the top of the page. This normally happens when there's something missing in the code.

I went into your site.html and noticed that you're missing all of the end of the page code. This happened to a few people so I think that someone had accidentally cut the out of the AA-template when we were first starting the project, and so a few people worked off that version.  It's an easy fix though.  All you have to do is copy and paste the code below at the end of your site.html (so after those last <p></p> tags) hit enter a few times to give yourself some room. Once you've pasted it in, commit it, then push origin, and double check to make sure that they work.  Then create another pull request to turn it in. Let me know if you have any other questions .

Jack

PS - On my end, it had me post the code between '' - make sure you delete the one before <br> and after</html> if they're there before you paste it into your code.


 `<br>

		 

		 


	</div>

       <br>
         <br>

   
    <div id="central-foot">


    
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <script src="../JS/bootstrap.min.js"></script>
	<script type="text/javascript">
		$(document).ready(function(){
			$("a[rel=citation]").popover({
				placement : 'top'
			}).click(function (p) {
				p.preventDefault();
				var $self = $(this);
				setTimeout(function () {
					$self.popover('hide');
				}, 4000);
			});
		});
	</script>

         
    <script src="../../centralize-nav-foot/sites-nav-foot.js"></script>

</body></html>`

**Comment by jabiggs13** [comment]
Hmm...now on my end it looked like it cut some of the code off. Here it is again just in case:

`<br>

		 

		 


	</div>

       <br>
         <br>

   
    <div id="central-foot">


    
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
    <script src="../JS/bootstrap.min.js"></script>
	<script type="text/javascript">
		$(document).ready(function(){
			$("a[rel=citation]").popover({
				placement : 'top'
			}).click(function (p) {
				p.preventDefault();
				var $self = $(this);
				setTimeout(function () {
					$self.popover('hide');
				}, 4000);
			});
		});
	</script>

         
    <script src="../../centralize-nav-foot/sites-nav-foot.js"></script>

</body></html>`

**Comment by CareoWin** [comment]
For some reason, I am un-able to fix the image pop-over coding.



  <figure style="float: right; width: 75%; padding-left: 20px; padding-top: 20px; padding-bottom: 20px;">

  

  <figcaption><a href="#" rel="citation" data-toggle="popover" data-content="Relief of Ptolemy I offering to Hathor. Photo by Museum of Fine Arts Boston, accessed 10 November 2018.">Image 4</a> - "Egyptian Ptolemaic Dynasty, reign of Ptolemy I Soter 305-282 B.C."</figcaption>

  </figure>


Cannot figure out what is wrong with it haha.

________________________________

**Comment by jabiggs13** [comment]
So the popover coding is correct, but they're not popping up because you're missing the last half of the code for the html page. Your html code ends with a citation wrapped in <p></p> tags. 

Since the code I pasted in isn't showing what it needs to, go in to the AA-template.html (https://github.com/msu-anthropology/daea-fs18/blob/master/sites/AA-template/AA-template.html) and scroll down to the bottom of the page - there will be the part where it says SITE NAME, AUTHOR, PERIOD, etc... - after that is BIG space until there is ending code starting with </div>. Copy this code including </div> all the way to the end of the code - it should be </body></html>.  Paste that code after your last bit of code in your html - the last <p>...</p> tags with your references in it. Commit and push that and see if that works. Comment back here once you do that and let me know how it goes.

**Comment by CareoWin** [comment]
It worked! Thank you so much for helping.


Caroline Nguyen

________________________________
